### PR TITLE
IGNITE-11556 Add color legend into series selection panel

### PR DESCRIPTION
**Pull request by AlexDel**
@Klaster1 I've added color mapping for series in series selector. Please have a look.

### CODE REVIEW

File: modules/web-console/frontend/app/components/ignite-chart-series-selector/style.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.bs-select-menu {
+  .bssm-item-text {
```
Comment (Klaster1) [inline modules/web-console/frontend/app/components/ignite-chart-series-selector/style.scss]: This rule applies to all menus across the app, is this what you intended?

File: modules/web-console/frontend/app/components/bs-select-menu/template.pug [hunk 2]
```diff
@@ -37,11 +37,11 @@ ul.bs-select-menu(
...
             ng-class=`{ 'bssm-item-button__active': $isActive($index) }`
             data-placement='right auto'
-            title='{{ ::match.label | bsSelectStrip }}'
+            title='{{ match.title || match.label | bsSelectStrip }}'
         )
             img.bssm-active-indicator.icon-left(
                 ng-src='{{ $isActive($index) ? "/images/checkbox-active.svg" : "/images/checkbox.svg" }}'
             )
-            span.bssm-item-text(ng-bind-html='match.label')
+            .bssm-item-text(ng-bind-html='$ctrl.trustHtml(match.label)')
```
Comment (Klaster1) [inline modules/web-console/frontend/app/components/bs-select-menu/template.pug]: Why `trustHtml` in menu template if you can do the same when building menu in ignite-chart-series-selector?

File: modules/web-console/frontend/app/components/bs-select-menu/strip.filter.js [hunk 3]
```diff
@@ -17,6 +17,9 @@
 
 export default function() {
     return function(val) {
-        return val ? val.replace(/(<\/?\w+>)/igm, '') : '';
+        if (typeof val !== 'string')
```
Comment (Klaster1) [inline modules/web-console/frontend/app/components/bs-select-menu/strip.filter.js]: Am I correct to assume an additional check is required in case you want to pass SCE'd content in?

File: modules/web-console/frontend/app/components/bs-select-menu/template.pug [hunk 4]
```diff
@@ -37,11 +37,11 @@ ul.bs-select-menu(
             ng-click='$select($index, $event); $event.stopPropagation();'
             ng-class=`{ 'bssm-item-button__active': $isActive($index) }`
             data-placement='right auto'
-            title='{{ ::match.label | bsSelectStrip }}'
```
Comment (Klaster1) [inline modules/web-console/frontend/app/components/bs-select-menu/template.pug]: If "sce in chart menu" approach works, consider reverting all unnecessary changes to bs-select-menu.

**Comment by dspavlov** [comment]
Closing PR for Closed issue. Feel free to reopen if it is necessary.
